tests/foreman/cli/test_host.py

```diff
@@ -590,30 +590,6 @@ def test_positive_katello_and_openscap_loaded():
         ), f'--{arg} not supported by update subcommand'
 
 
-@pytest.mark.cli_host_create
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_positive_register_with_no_ak(
-    module_lce, module_org, module_promoted_cv, rhel7_contenthost, target_sat
-):
-    """Register host to satellite without activation key
-
-    :id: 6a7cedd2-aa9c-4113-a83b-3f0eea43ecb4
-
-    :expectedresults: Host successfully registered to appropriate org
-
-    :parametrized: yes
-
-    :CaseLevel: System
-    """
-    rhel7_contenthost.install_katello_ca(target_sat)
-    rhel7_contenthost.register_contenthost(
-        module_org.label,
-        lce=f'{module_lce.label}/{module_promoted_cv.label}',
-    )
-    assert rhel7_contenthost.subscribed
-
-
 @pytest.mark.cli_host_create
 @pytest.mark.tier3
 def test_negative_register_twice(module_ak_with_cv, module_org, rhel7_contenthost, target_sat):
```

Comment:
Can we consider moving this to cli/test_registration?

Reply:
Move to registration module.